### ISSUE TITLE
No audio

### ISSUE THREAD

**Issue by gigjacker** [westlicht/rc505-editor#5]
I've loaded a backup of my RC505 and I can see all the tracks as I expect but when I hit play, no sound is coming out.  There doesn't seem to be a way to select an output device so I assume it's using the Windows 10 default.

**Comment by codeCoaster** [comment]
Hi, did you ever resolve this issue?

**Comment by westlicht** [comment]
No, I haven't worked on this for ages. It's open source, somebody can pick it up. First thing would probably be to update to latest version of JUCE. I know they have a premade settings window, so that should be fairly easy to add.
